docs/src/czt.md

```diff
@@ -0,0 +1,11 @@
+# CZTs
+Chirp Z Transformations: Allows Fourier-transformation and at the same time zooming into the result,
+which is why it is also called the Zoomed-FFT algorithm.
+The algorithm is based on a publication [Rabiner, Schafer, Rader, The Cirp z-Transform Algorithm, IEEE Trans AU 17(1969) p. 86] and a matlab Version written by the group of H. Gross. It currently needs three FFTs to perform its work.
```

Comment:
what is the license of the Matlab source code? Might break our license.

Reply:
I asked him and he agreed that I can use it in Julia and release my version under the MIT license, as far as I remember.

Reply:
I updated the description to reflect the consent of H. Gross (28.10.2020).